### ISSUE TITLE
"Style" Plugin needs more HTML attribute options

### ISSUE THREAD

**Issue by bartmika** [divio/djangocms-style#19]
I am looking to add a "Style" plugin so I can write the following code:

```
  <div class="container-fluid">
```

But when I look in the **Advanced Settings** I don't see any ways to add the particular HTML attribute.
![screen shot 2015-11-26 at 10 32 58 am](https://cloud.githubusercontent.com/assets/5767602/11426327/33a20f80-9429-11e5-9530-08d206ef27fd.png)

I would like to open up a ticket to allowm more HTML attribute options to be available for the **Style Plugin**, more specifically:
- id
- class
- style
- onclick

**Comment by bartmika** [comment]
Here's another example. I want ``<article class="text-center">`` so I:

Step 1: Add **Style** placeholder and fill in classes.
![screen shot 2015-11-26 at 4 36 36 pm](https://cloud.githubusercontent.com/assets/5767602/11430958/ba7bd3c8-945c-11e5-9b0c-419e27165a44.png)

Step 2: Then refresh page after saving and I don't see the **class** field being updated:
![screen shot 2015-11-26 at 4 37 27 pm](https://cloud.githubusercontent.com/assets/5767602/11430976/e547acee-945c-11e5-9f82-4dfeeabc5e4c.png)

So in summary, if you can grant the **style plugin** placeholder to allow more HTML attributes that would be great!

**Comment by FinalAngel** [comment]
@bartmika thank you for the issue report, I'll have a look at it!

**Comment by FinalAngel** [comment]
fixed in https://github.com/divio/djangocms-style/pull/25
